src/NRedisStack/Search/FTSpellCheckParams.cs

```diff
@@ -0,0 +1,103 @@
+using NRedisStack.Search.Literals;
+namespace NRedisStack.Search
+{
+    public class FTSpellCheckParams
+    {
+        List<object> args = new List<object>();
+        private List<KeyValuePair<string, string>>? terms = null;
```

Comment:
Why this has default value as `null`? Shouldn't be `new List<KeyValuePair<string, string>>()`